src/test/java/com/google/firebase/auth/FirebaseAuthIT.java

```diff
@@ -625,6 +625,45 @@ public void testVerifyIdToken() throws Exception {
     auth.deleteUserAsync("user2");
   }
 
+  @Test
+  public void testVerifyIdTokenUserDisabled() throws Exception {
+    String customToken = auth.createCustomToken("disabledUser");
```

Comment:
Nit: It's often safer to use random UID strings instead of static ones (in case 2 developers were running the tests in parallel against a shared project). You can use this API to generate random UIDs:

```java
RandomUser randomUser = UserTestUtils.generateRandomUserInfo();
String customToken = auth.createCustomToken(randomUser.getUid());
```